apps/web-staking/src/app/components/redeem/History.tsx

```diff
@@ -16,8 +16,10 @@ import { MODAL_BODY_TEXT } from "./Constants";
 import { WriteFunctions, executeContractWrite } from "@/services/web3.writes";
 import { BaseModal, PrimaryButton } from "@/app/components/ui";
 import { TextButton } from "@/app/components/ui/buttons";
-import { useBlockIp, useGetKYCApprovedForRedemptionClaim } from "@/app/hooks";
+import { useBlockIp, useGetKYCApproved, useGetKYCApprovedForRedemptionClaim } from "@/app/hooks";
```

Comment:
Why is `useGetKYCApproved` added here ?

Reply:
It was an unused import, has been removed.